### PR TITLE
Taylor- Creating signup component

### PR DESCRIPTION
**Pull request by TaylorDJones11**
## What type of PR is this? (check all applicable)

- [ ] Feature
- [ ] Refactor
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update
- [x] Other: Creating Signup Component 
  
  
## Description of what this Pull Request does
- The Signup Component is styled with Tailwind to take in the user's information: First & Last name, Email, Username, Password, and password confirmation. 
- The Signup Component  will allow the user to create an account using Google, Github or through Email

## Related Tickets & Documents
- Closes https://github.com/annonymousauthority/P2PCoder/issues/37

  
## QA Instructions, Screenshots, Recordings
Screenshot 2023-05-20 at 15 04 33

### CODE REVIEW

**Comment by TaylorDJones11** [comment]
- I'm having issues with finding the correct spacing between the columns using flex. 
- I don't know where to find the icon components

File: src/components/signup_comp.js [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+        </button>
+      </div>
+
+      <div className="flex items-center text-center">
+        <div className="border-b-2 border-gray-300 w-full relative flex justify-center"></div>
+        <p className="text-black w-1/6">OR</p>
+        <div className="border-b-2 border-gray-300 w-full relative flex justify-center"></div>
+      </div>
+
+      {/* Signup Form */}
```
Comment (annonymousauthority) [inline src/components/signup_comp.js:28]: No need for comments

**Review by annonymousauthority — CHANGES_REQUESTED** [review]

**Comment by TaylorDJones11** [comment]
I implemented the changes you suggested at the meeting and this is the outcome 😭


Screenshot 2023-05-19 at 18 02 45

**Comment by TaylorDJones11** [comment]
Another question @annonymousauthority  when I change anything in the parent div, why does it never affect the first child div but the following ones? 

Screenshot 2023-05-20 at 10 10 21

**Comment by Remi-dee** [comment]
Hi @TaylorDJones11 hoping you’ve solved these?
However I’ll be willing to help, if you need any.

**Review by annonymousauthority — APPROVED** [review]
Well done. LGTM

**Pull request merged**